vue-frontend/src/components/home-new/ContributionSection.vue

```diff
@@ -0,0 +1,196 @@
+        </div>
+        <div
+          :class="[
+            isMobile
+              ? 'tw-absolute tw-mt-[135px] tw-w-[12.563rem] tw-left-[170px]'
+              : ' ',
+          ]"
+          class="sm:tw-flex sm:tw-flex-col tw-basis-[35%] md:tw-basis-2/5 tw-my-auto tw-bg-[#FFFFFF] tw-p-[0.5rem] sm:tw-p-5 lg:tw-p-[2rem] v2-title-3-text tw-shadow-[0_0px_10px_rgba(0,0,0,0.1)]"
+        >
+          <div>
+            <div
+              @touchstart.passive="activeIndex = index"
+              :class="index % 2 != 0 ? 'sm:tw-flex-row-reverse' : ''"
+              class="tw-flex tw-flex-row tw-justify-between tw-items-center"
+            >
+              <div>
+                <span
+                  class="tw-w-[1.25rem] md:tw-w-[4.75rem] tw-h-[0.625rem] md:tw-h-[1.875rem] tw-from-[#ff835b] tw-to-[#f2709c] tw-bg-gradient-[270.11deg] tw-p-[2px] md:tw-p-[4px] tw-rounded-full tw-inter-semibold tw-text-[0.625rem] md:tw-text-[1rem] tw-leading-[2.5625rem] md:tw-leading-[1.5rem] tw-text-white"
+                >
+                  <font-awesome-icon
+                    class="fa tw-w-[10px] md:tw-w-[14px] tw-h-[10px] md:tw-h-[14px] tw-px-[5px] tw-text-white"
+                    icon="star"
+                  />{{ contribution.starPoints }}</span
+                >
+              </div>
+              <div>
+                <div>
+                  <a
+                    class="tw-flex tw-items-center tw-font-inter-bold tw-text-[0.75rem] md:tw-text-[1.25rem] tw-leading-[2.5625rem] tw-duration-300 tw-ease-in-out hover:tw-scale-[1.1] tw-hover-hover"
+                    :href="contribution.link"
+                    target="_blank"
+                    @click.native="$gtag.event('tap_library')"
+                  >
+                    <font-awesome-icon
+                      class="tw-mr-2.5 tw-w-[16px] md:tw-w-[30px] lg:tw-w-10 tw-h-[16px] md:tw-h-[30px] lg:tw-h-10 tw-text-pink-300"
+                      :icon="icon"
+                    />
+                    <div class="v2-canopas-gradient-text">GitHub</div>
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div>
+              <h1
+                class="tw-mt-[20px] md:tw-mt-[41px] tw-font-inter-semibold tw-text-[1rem] md:tw-text-[1.875rem] tw-leading-[1.3rem] md:tw-leading-[2.4375rem] tw-text-black-core/[0.87]"
+              >
+                {{ contribution.title }}
+              </h1>
+            </div>
+            <div class="tw-mt-5">
+              <span
+                class="tw-hidden md:tw-block tw-font-inter-light tw-text-[1.5rem] tw-leading-[1.875rem] tw-text-black-core/[0.6]"
+                >{{ contribution.description }}</span
+              >
+            </div>
+            <div
+              @touchstart.passive="activeIndex = index"
+              class="tw-flex tw-flex-row tw-justify-between tw-mt-[10px] md:tw-mt-[30px] tw-font-inter-regular tw-text-[1rem] tw-leading-[1.5rem] tw-text-black-core/[0.87]"
```

Comment:
Font size is 10px in mobile. Also, the design is the same as figma on mobile devices. Check design and all the sizes for the whole section